spec/task_005_lib_spec.rb

```diff
@@ -0,0 +1,160 @@
+require './lib/task_005_lib'
+
+RSpec.describe 'parse_input(input_data)' do
+  it ' parses input data ' do
+    input = <<~INPUT
+      0,9 -> 5,9
+      8,0 -> 0,8
+      9,4 -> 3,4
+    INPUT
+
+    lines = Task005.parse_input(input)
+
+    expect(lines[0]).to have_attributes(
+      point1: have_attributes(x: 0, y: 9),
+      point2: have_attributes(x: 5, y: 9),
+    )
+    expect(lines[1]).to have_attributes(
+      point1: have_attributes(x: 8, y: 0),
+      point2: have_attributes(x: 0, y: 8),
+    )
+    expect(lines[2]).to have_attributes(
+      point1: have_attributes(x: 9, y: 4),
+      point2: have_attributes(x: 3, y: 4),
+    )
```

Comment:
Great job. You can go even further now and merge it into a single `expect` using another RSpec matcher https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/contain-exactly-matcher :)

Reply:
I try this with `to eq`, doesn't work, now works :)